xml/gl.xml

```diff
@@ -2734,9 +2734,9 @@ typedef unsigned int GLhandleARB;
         <enum value="0x84F2" name="GL_ALL_COMPLETED_NV" group="FenceConditionNV"/>
         <enum value="0x84F3" name="GL_FENCE_STATUS_NV" group="FenceParameterNameNV"/>
         <enum value="0x84F4" name="GL_FENCE_CONDITION_NV" group="FenceParameterNameNV"/>
-        <enum value="0x84F5" name="GL_TEXTURE_RECTANGLE" group="CopyImageSubDataTarget,TextureTarget"/>
-        <enum value="0x84F5" name="GL_TEXTURE_RECTANGLE_ARB"/>
-        <enum value="0x84F5" name="GL_TEXTURE_RECTANGLE_NV"/>
+        <enum value="0x84F5" name="GL_TEXTURE_RECTANGLE" group="CopyImageSubDataTarget,TextureTarget,EnableCap,GetPName"/>
+        <enum value="0x84F5" name="GL_TEXTURE_RECTANGLE_ARB" group="TextureTarget,EnableCap,GetPName"/>
+        <enum value="0x84F5" name="GL_TEXTURE_RECTANGLE_NV" group="TextureTarget,EnableCap,GetPName"/>
```

Comment:
Same comment as for `GL_TEXTURE_RECTANGLE`.

Adding `TextureTaget` here is good.

Reply:
From ARB_texture_rectangle

     10) How are rectangular textures enabled?

     Rectangular textures are enabled by enabling the
     GL_TEXTURE_RECTANGLE_ARB texture target via glEnable
     (GL_TEXTURE_RECTANGLE_ARB). This enable is prioritized above
     GL_TEXTURE_2D and below GL_TEXTURE_3D.

     From lowest priority to highest priority: GL_TEXTURE_1D,
     GL_TEXTURE_2D, GL_TEXTURE_RECTANGLE_ARB, GL_TEXTURE_3D,
     GL_TEXTURE_CUBE_MAP.

     Accepted by the <cap> parameter of Enable, Disable and IsEnabled;
     by the <pname> parameter of GetBooleanv, GetIntegerv, GetFloatv
     and GetDoublev; and by the <target> parameter of BindTexture,
     GetTexParameterfv, GetTexParameteriv, TexParameterf, TexParameteri,
     TexParameterfv and TexParameteriv:

    TEXTURE_RECTANGLE_ARB            0x84F5

From NV_texture_rectangle

     How are rectangular textures enabled?

      RESOLUTION:  Since rectangular textures add a new texture target,
      you enable rectangular textures by enabling this target.  Example:

        glEnable(GL_TEXTURE_RECTANGLE_NV);

     Accepted by the <cap> parameter of Enable, Disable, IsEnabled, and
     by the <pname> parameter of GetBooleanv, GetIntegerv, GetFloatv,
     and GetDoublev, and by the <target> parameter of BindTexture,
     GetTexParameterfv, GetTexParameteriv, TexParameterf, TexParameteri,
     TexParameterfv, and TexParameteriv:

        TEXTURE_RECTANGLE_NV               0x84F5